styles/styles.css

```diff
@@ -432,6 +432,57 @@ main .section.full-width>.section-container {
   padding-right: 0;
 }
 
+/* START section - brand detail page */
```

Comment:
I'm wondering, if these sections, which you neatly marked with START/END only apply to the brands page, would it make sense to put them in a page-specific style? I know it was said that we'd like to maybe avoid page-specific styles but maybe it's better than putting styles that apply to only 1 page in the general styles.css?

Reply:
@bosschaert Yeah, it's pretty page-level styling. I had the same question, but as we discussed in the standup meeting, the page-level style won't be used in sunstar.com site.
@sdmcraft Shall we revive page-level styling?

Reply:
Can't we use general styles in place of page level styles. I think we agreed over one approach that if we encounter such kind of cases, we will ask Dejan regarding that case.

Reply:
I was just wondering how far does `narrow` section style take us on this. [Here's what I see](https://main--sunstar--hlxsites.hlx.page/_drafts/satyam/document). Looks somewhat similar to live but not quite there. I think the images should be centered and margin/paddings should be adjusted. But likely the `narrow` section variant itself can benefit from these.

Reply:
I just tried to use `narrow` but it doesn't look perfect to me. To reproduce the original site, if I add the styling to `narrow`, I think it will impact other pages such as image styling. Any suggestion in this case? For now, I added it separately into style.css.

narrow version:
https://issue-30--sunstar--hlxsites.hlx.page/_drafts/jiang/brands/consumer-health-beauty-withnarrower

Reply:
I see that you use the `narrower` style right now, the differences I personally see are:

- The `kendojo` and `equitance` logo images are not centred and a bit bigger, I think that you can add two variants to the image collage, something like `zoomed-in` and `centred` to achieve the live website size and positioning
- The rest of the images are smaller; I think you can add a section style called something like `narrow-pictures` that sets the img widths
- The text is centred, not left-aligned; I think another section style called something like `left-aligned-text` would do the trick; either this or a variation of the `text` block that does the same

Just my two cents

Reply:
Thanks @dnbute! It's an approach that I thought of before but I'd like to reduce the authoring effort by avoiding adding more variants to blocks, that's why I put all staff into one metadata.
 
It looks like a maze game, @sdmcraft  @bosschaert  @jindaliiita any thoughts?

Reply:
@sdmcraft @bosschaert @jindaliiita @dnbute Okay, I moved out the styling from styles.css and added them to blocks. Please help to review.
https://issue-30--sunstar--hlxsites.hlx.page/_drafts/jiang/brands/consumer-health-beauty